src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -294,6 +305,16 @@ class CompilerContainer {
     }
   }
 
+  _updateLanguageSelector () {
+    if (this._retriveVersion() < '0.5.7') {
```

Comment:
how this can work? you should probably use semver here.

Reply:
:) it works. ok I'll look at it